FuelSDK-Test/RefreshTokenTest.cs

```diff
@@ -0,0 +1,36 @@
+﻿using NUnit.Framework;
+using System;
+using FuelSDK;
+using Newtonsoft.Json.Linq;
+
+namespace FuelSDK.Test
+{
+    [TestFixture()]
+    public class RefreshTokenTest
+    {
+        ETClient client;
+ 
+        [OneTimeSetUp]
+        public void Setup()
+        {
+            client = new ETClient();
+        }
+
+        [Test()]
+        public void Oauth2RefreshToken()
```

Comment:
The test name could be made more explicit like RefreshTokenWithOauth2_WhenForcingARefresh_GetsNewTokenAndRefreshToken

Reply:
That will make the method name too lengthy. Also when you have multiple asserts, is it not a better way to handle the messages in the assert statement. However i have changed the message name a little bit.